span-stacktrace/README.md

```diff
@@ -20,26 +20,38 @@ section below to configure it.
 ### Manual SDK setup
 
 Here is an example registration of `StackTraceSpanProcessor` to capture stack trace for all
-the spans that have a duration >= 1000 ns. The spans that have an `ignorespan` string attribute
+the spans that have a duration >= 1 ms. The spans that have an `ignorespan` string attribute
 will be ignored.
 
 ```java
 InMemorySpanExporter spansExporter = InMemorySpanExporter.create();
 SpanProcessor exportProcessor = SimpleSpanProcessor.create(spansExporter);
 
+Map<String, String> configMap = new HashMap<>();
+configMap.put("otel.span.stacktrace.min.duration", "1ms");
```

Comment:
```suggestion
configMap.put("otel.java.experimental.span-stacktrace.min.duration", "1ms");
```

Reply:
Is this really needed?

Reply:
it's only necessary if you want to override the default, but that's optional if you think it's simpler let's remove it.

Reply:
Sorry I completely misread the discussion here, so please ignore my previous msg.

There are quite a few other config options patterns in the contrib repo (non-exhaustive list)
- `otel.jmx.*` used by the current JMX gatherer, but unlikely to conflict with anything due to the nature of the gatherer.
- `otel.inferred.spans.*` for inferred spans, which means we would have to migrate those for consistency
- `otel.java.experimental.*` pattern used only by the baggage to span attribute module until we also use it here.

So it seems that so far the `otel.<feature>` where `<feature>` somehow matches the module name is the most common option.

What I don't like with the `experimental` is that is puts a "stability/maturity" semantic into the configuration. For example, the inferred spans are quite mature because we migrated it from our own agent.

Reply:
There are rules defined or about to be defined in semantic conventions for attribute and metric names, could we make most config options also fit the existing ones ?

For example, despite the shared ownership of contrib repo, would it make sense to include those in the documentation for Java https://opentelemetry.io/docs/languages/java/configuration/, it already contains:
- general SDK configuration with implementation details for Java with `otel.` prefix
- Java specific configuration for the SDK and Instrumentation (with the `otel.java.` prefix

With that scheme, I think that maybe having a `otel.java.<feature>.` pattern as prefix for contrib would be nice because:
- we can have a clear list of features weither they are in contrib or not (with appropriate links)
- from the person in charge of configuration, the fact it's part of contrib is an implementation detail, if the feature is later moved to a separate artifact is not relevant to how it's configured.

Reply:
After discussing this on our side (Elastic), we think we should separate this discussion about config and feature maturity from this PR, we will follow-up in future Java SIG meetings.

As a consequence, for now we'll just update it to `otel.java.experimental.span-stacktrace.min.duration` and we can enhance it later.